### ISSUE TITLE
Render vehicle=destination etc.

### ISSUE THREAD

**Issue by sanderd17** [gravitystorm/openstreetmap-carto#1312]
Currently, the style seems to render only access=*, not any of the sub categories.

When you want to tag that a road is impassable for a vehicle, but a pedestrian can pass, there are multiple ways of doing that:

Two of the options include

```
vehicle=no
```

and 

```
access=no
foot=yes
horse=yes
```

Since the main style only takes the access=* key into account, both ways of tagging result in different rendering. This results in some mappers chosing one variant over the other, just because they want or don't want to see that rendering (a.k.a. tagging for the renderer). On the other hand, when you see the no-access rendering, you're not sure for which modes of transport it counts.

To solve this, I think that the carto style must take one preferred vehicle, and use the entire hierarchy (http://wiki.openstreetmap.org/wiki/Key:access#Land-based_transportation) of access tags to follow that vehicle. I guess "motorcar" is the most obvious choice for a preferred vehicle.

So to summarise, the style should probably do the following

* If motorcar=* is present, use it
* else, if motor_vehicle=*is present, use it
* else if vehicle=* is present, use it
* else if access=* is present, use it

With that style, we will be at least sure that the rendering counts for regular cars. As it's impossible to combine all modes of transport into a single rendering, I guess this is the best that can be achieved.

**Comment by math1985** [comment]
Duplicate with #214.
